### PR TITLE
Flatsat merge

### PR DESCRIPTION
**Pull request by nsorense4**

### CODE REVIEW

**Review by yunqingyi — APPROVED** [review]

File: hardware_interface/source/eps.c [hunk 1]
```diff
@@ -190,6 +190,10 @@ void EPS_getHK(eps_instantaneous_telemetry_t *telembuf, eps_startup_telemetry_t
 eps_mode_e get_eps_batt_mode() {
     eps_refresh_instantaneous_telemetry();
     eps_instantaneous_telemetry_t eps = get_eps_instantaneous_telemetry();
+
+    //Adding these in order to get the startup telemetry - should be added elsewhere
```
Comment (Taylorr82) [inline hardware_interface/source/eps.c]: Where should it be instead?

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

**Comment by Taylorr82** [comment]
When doing commit messages keep them in imperative form, and no "we". We know who made the commit, the imperative form is actually from the perspective of the commit

File: hardware_interface/source/eps.c [hunk 2]
```diff
@@ -190,6 +190,10 @@ void EPS_getHK(eps_instantaneous_telemetry_t *telembuf, eps_startup_telemetry_t
 eps_mode_e get_eps_batt_mode() {
     eps_refresh_instantaneous_telemetry();
     eps_instantaneous_telemetry_t eps = get_eps_instantaneous_telemetry();
+
+    //Adding these in order to get the startup telemetry - should be added elsewhere
+    eps_refresh_startup_telemetry();
+    eps_startup_telemetry_t eps_startup = get_eps_startup_telemetry();
```
Comment (arrooney) [inline hardware_interface/source/eps.c]: I'm not sure I follow why this is here

File: hardware_interface/source/eps.c (hunk 1 above)
Comment (nsorense4) [reply hardware_interface/source/eps.c]: I've added them to the housekeeping service - this should be where they belong.

File: hardware_interface/source/eps.c (hunk 2 above)
Comment (nsorense4) [reply hardware_interface/source/eps.c]: They weren't called anywhere else. I've removed them.

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**